libraries/OPAMP/src/OPAMP.h

```diff
@@ -0,0 +1,48 @@
+#ifndef _OPAMP_H_
+#define _OPAMP_H_
+
+#include <stdint.h>
+
+/* Available speed modes */
+/* Note: No middle speed mode on the Uno R4 boards */
+enum OpampSpeedMode {
+    OPAMP_SPEED_LOWSPEED = 0,
+    OPAMP_SPEED_HIGHSPEED = 1,
+};
+
+/* The supported boards have 4 OPAMP channels, however, only channel 0 is accessible. */
+/* All other channels are connected to the LED matrix or not exposed. */
+#define ARDUINO_USED_OPAMP_CHANNEL 0

```

Comment:
That's a bit harsh. How about allowing selection of the desired op amp channel in `begin`, and overload the `begin` method with the default parameter?

What if someone makes his own design and wants to use another channel? What if Arduino creates a new board that supports multiple op amp channels? Hardcoding it like this creates a liability down the line.

Reply:
Indeed this is currently hardcoded. Should I make the opamp channel number a parameter for `begin()` with default value = 0? Then it would be less confusing for Arduino board users.

Reply:
Yes, please do so. However, I prefer function overloading over default parameters, just provide two different `begin()` methods.

Reply:
* Macro renamed to be `ARDUINO_DEFAULT_OPAMP_CHANNEL`
* Added overloaded `begin()` function, one without channel mask (will activate default channel 0), one with explicit channel mask
* Added pins for all 4 channels in constant data structure
* Symettrically added overloaded `end()` function, one without parameters stopping all OPAMP channels, one with a channel mask stopping only the specified channels in the bitmask